rampwf/score_types/detection.py

```diff
@@ -144,14 +144,9 @@ def __init__(self, name=_name, precision=2, iou_threshold=0.5):
         self.iou_threshold = iou_threshold
 
     def __call__(self, y_true, y_pred):
-        y_pred_conf = [detected_object[0]
-                       for single_detection in y_pred
-                       for detected_object in single_detection]
-        min_conf, max_conf = np.min(y_pred_conf), np.max(y_pred_conf)
-        conf_thresholds = np.linspace(min_conf, max_conf, 20)
-        ps, rs = precision_recall_curve(y_true, y_pred, conf_thresholds,
-                                        iou_threshold=self.iou_threshold)
-        return average_precision_interpolated(ps, rs)
+        ps, rs = precision_recall_curve_greedy(
+            y_true, y_pred, iou_threshold=self.iou_threshold)
+        return average_precision_exact(ps, rs)
```

Comment:
`precision_recall_curve_greedy` returns 3 objects so it should be `_, ps, rs = ...`

Reply:
yep